manim/utils/testing/_frames_testers.py

```diff
@@ -16,15 +18,15 @@
 
 
 class _FramesTester:
-    def __init__(self, file_path: Path, show_diff=False) -> None:
+    def __init__(self, file_path: Path, show_diff: bool = False) -> None:
         self._file_path = file_path
         self._show_diff = show_diff
         self._frames: np.ndarray
         self._number_frames: int = 0
         self._frames_compared = 0
 
     @contextlib.contextmanager
-    def testing(self):
+    def testing(self) -> Generator[Any, Any, Any]:
```

Comment:
Wouldn't this be
```suggestion
    def testing(self) -> Generator[None, None, None]:
```
since it yields nothing?